x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/components/precision_slider/precision_slider.test.tsx

```diff
@@ -59,11 +60,14 @@ describe('PrecisionSlider', () => {
 
   describe('Step Description', () => {
     it('is visible when there is a step description', () => {
-      setMockValues({ ...MOCK_VALUES, precision: 10 });
+      setMockValues({
+        ...MOCK_VALUES,
+        searchSettings: { ...MOCK_VALUES.searchSettings, precision: 10 },
+      });
       rerender(wrapper);
 
       expect(wrapper.find('[data-test-subj="StepDescription"]').render().text()).toEqual(
-        'Default. High recall, low precision.'
```

Comment:
Luckily(????) I had used the default text (step 2) instead of the description text for step 10 here.